apps/dashboard/app/models/project.rb

```diff
@@ -89,25 +88,22 @@ def metadata
     manifest.metadata
   end
 
-  def manifest_path
-    File.join(config_dir, 'manifest.yml')
+  def title
+    manifest.metadata[:title]
   end
```

Comment:
Manifests don't have a `title`. They have a `name` which we can titleize.

Seems we're starting to collide with the `project#name`?

Reply:
So the `title` is called `name` in the `Manifest` class? 

That still doesn't return the manifest title string when I tried `manifest.name` but `manifest.description` works fine so maybe I'm misunderstanding how to get the `title` out of the `manifest.yml`.

Reply:
Look at this through an MVC lens.  What do you have vs. how you show it.  I guess we're using the phrase or attribute `title` to mean `name`.

I'm not sure why you can't get anything out of `manifest#name` but you should. That's what ood_app does.
https://github.com/OSC/ondemand/blob/5f8b439eb46e0b42f8ee02227e93928042db920b/apps/dashboard/app/apps/ood_app.rb#L40-L42

Also - think about simplicity: How many different attributes to we need for "the simple human readable name for the project".  I mean, do we really need 2 different things, name and title?